apps/datahub/src/app/home/home-header/home-header.component.html

```diff
@@ -16,32 +16,31 @@
         class="text-[18px]"
         (itemSelected)="onFuzzySearchSelection($event)"
       ></gn-ui-fuzzy-search>
-      <div class="flex h-0 py-5" [style.opacity]="-0.6 + expandRatio * 2">
+      <div class="flex h-0 py-5 gap-3" [style.opacity]="-0.6 + expandRatio * 2">
         <datahub-header-badge-button
           [routerLink]="ROUTE_SEARCH"
-          class="mr-3"
           *ngIf="isAuthenticated$ | async"
           icon="star"
           label="datahub.header.myfavorites"
           [toggled]="searchFacade.favoritesOnly$ | async"
           (action)="listFavorites($event)"
         ></datahub-header-badge-button>
-        <datahub-header-badge-button
-          class="mr-3"
-          label="datahub.header.lastRecords"
-          [toggled]="
-            (searchFacade.sortBy$ | async) === SORT_BY_PARAMS.CREATE_DATE
-          "
-          (action)="clearSearchAndSort($event, SORT_BY_PARAMS.CREATE_DATE)"
-        ></datahub-header-badge-button>
-        <datahub-header-badge-button
-          class="mr-3"
-          label="datahub.header.popularRecords"
-          [toggled]="
-            (searchFacade.sortBy$ | async) === SORT_BY_PARAMS.USER_SAVED_COUNT
-          "
-          (action)="clearSearchAndSort($event, SORT_BY_PARAMS.USER_SAVED_COUNT)"
-        ></datahub-header-badge-button>
+        <div>
+          <button
+            class="badge-btn bg-primary-opacity-25"
+            (click)="clearSearchAndSort(SORT_BY_PARAMS.CREATE_DATE)"
+          >
+            <span translate="">datahub.header.lastRecords</span>
+          </button>
+        </div>
+        <div>
+          <button
+            class="badge-btn bg-primary-opacity-25"
+            (click)="clearSearchAndSort(SORT_BY_PARAMS.POPULARITY)"
+          >
+            <span translate="">datahub.header.popularRecords</span>
+          </button>
+        </div>
```

Comment:
I'm not comfortable having "low level" buttons in a complex component like that (especially since now their styling is specified in the datahub CSS). I think here it would make sense to use `gn-ui-button` at least.

Also these buttons are missing the `type="button"` attribute as well as a title (good examples on why using a common button component helps with accessibility).

Reply:
Good point, I completely short cut this part.
I will use a gn-ui-button and see if the style is handled properly.

BTW, I would like to rise a point, TW with apply is also a way to define components, at least ui components. We face the overlap in Rennes3D project, where the question is "shall we use a tailwind component or a Vue component for this purpose". That's why i am not completly unfavored to global css TW component.

To be discussed

Reply:
Thinking about it again, it appears to me that `gn-ui-button` is almost useless.
Instead of passing a `type` then coding thousands of stuff to style the button, to propagate the event (which is useless btw) and so on, simple TW components like `btn-primary` would have given the same result in a cleaner way (IMO).
At the very least, the `gn-ui-button` could be a proxy to a button with a TW component class (@apply).
To me, all of this should be revamped.

Also for the datahub, there is absolutely no consistency in the styling of the components (buttons, panels, cards, animations, shadow etc...), and TW components are the answer for this IMO. It's way easier to define them instead of tweaking Angular components for styling purposes.

I would add a task for that within the current contract, and keep this PR as it is for now.
Would it be acceptable ?

Reply:
Sure! Indeed there are probably many improvements to be done to the buttons, let's talk about that later on.